composer.json

```diff
@@ -10,14 +10,14 @@
   },
   "require": {
     "php": ">=7.3",
-    "illuminate/http": "^8.0 || ^9.0",
+    "illuminate/http": "^9.0 || ^10.0",
     "nyholm/psr7": "^1.2",
     "psr/http-server-middleware": "^1.0",
     "symfony/psr-http-message-bridge": "^2.0"
   },
   "require-dev": {
-    "friendsofphp/php-cs-fixer": "^2.16",
-    "phpunit/phpunit": "^9.0"
+    "friendsofphp/php-cs-fixer": "^3.0",
```

Comment:
This upgrade require changes in configuration.

```
- Configuration file `.php_cs` is deprecated, rename to `.php-cs-fixer.php`.
- Option "importsOrder" is deprecated, use "imports_order" instead.
- Option "sortAlgorithm" is deprecated, use "sort_algorithm" instead.
- PhpCsFixer\Config::create is deprecated since 2.17 and will be removed in 3.0, use the constructor instead.
- Rule "psr0" is deprecated. Use "psr_autoloading" instead.
- Rule "psr4" is deprecated. Use "psr_autoloading" instead.
- Rule "trailing_comma_in_multiline_array" is deprecated. Use "trailing_comma_in_multiline" instead.
```